newswires/client/src/WireDetail.tsx

```diff
@@ -19,50 +22,67 @@ export const WireDetail = ({ wire }: { wire: WireData }) => {
 			: undefined;
 	}, [wire]);
 
-	const listItems = [
-		{
-			title: 'Byline',
-			description: byline ?? 'Not found',
-		},
-		{
-			title: 'Keywords',
-			description: (
-				<EuiFlexGroup wrap responsive={false} gutterSize="xs">
-					{keywords?.map((keyword) => (
-						<EuiFlexItem key={keyword} grow={false}>
-							<EuiBadge color="primary">{keyword}</EuiBadge>
-						</EuiFlexItem>
-					))}
-				</EuiFlexGroup>
-			),
-		},
-		{
-			title: 'Usage restrictions',
-			description: usage ?? 'Not found',
-		},
-		{
-			title: 'Body text',
-			description: safeBodyText ? (
-				<article dangerouslySetInnerHTML={{ __html: safeBodyText }} />
-			) : (
-				'Not found'
-			),
-		},
-	];
+	const nonEmptyKeywords = useMemo(
+		() => keywords?.filter((keyword) => keyword.trim().length > 0) ?? [],
+		[keywords],
+	);
 
 	return (
 		<Fragment>
-			<h3
-				css={css`
-					font-weight: 300;
-				`}
-			>
-				{wire.content.subhead}
-			</h3>
+			<EuiScreenReaderLive focusRegionOnTextChange>
+				<h3
+					css={css`
+						font-weight: 300;
+					`}
+				>
+					{wire.content.subhead}
+				</h3>
+			</EuiScreenReaderLive>
 
 			<EuiSpacer size="m" />
 
-			<EuiDescriptionList listItems={listItems} />
+			<EuiDescriptionList>
+				{byline && (
+					<>
+						<EuiDescriptionListTitle>Byline</EuiDescriptionListTitle>
+						<EuiDescriptionListDescription>
+							{byline}
+						</EuiDescriptionListDescription>
+					</>
+				)}
+				{nonEmptyKeywords.length > 0 && (
+					<>
+						<EuiDescriptionListTitle>Keywords</EuiDescriptionListTitle>
+						<EuiDescriptionListDescription>
+							<EuiFlexGroup wrap responsive={false} gutterSize="xs">
+								{nonEmptyKeywords.map((keyword) => (
+									<EuiFlexItem key={keyword} grow={false}>
```

Comment:
annoyingly, agencies are bad at deduplicating keywords...
<img width="637" alt="image" src="https://github.com/user-attachments/assets/6df6df20-8592-4988-b7f8-a83b3eb2a323">
we might want to do that deduplication for them, either at ingestion or rendering (there's no point in showing "general news" twice)

Reply:
good shout! I've opened a PR here for the ingestion-time changes: https://github.com/guardian/newswires/pull/44